src/pool/poolClient.ts

```diff
@@ -787,6 +787,9 @@ export class Client {
 
     let rateModel: acrossConfigStore.RateModel | undefined = undefined;
     try {
+      // Use the default rate model (i.e. not any of the routeRateModels to project the Pool's APR). This assumes
+      // that the default rate model is the most often used, but this may change in future if many different
+      // route rate models are set.
```

Comment:
@daywiss you wrote this logic originally, now that we support deposit-path specific rate models, are we still ok using as the projected APR? @mrice32 @pxrl wdyt

Reply:
I think we want to get rid of projected APR anyway. I think this is fine for now and still a decent approximation since it uses the default rate model.